lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -310,6 +314,12 @@ private JourneyResponse getJourneyResponse(
                             vcs,
                             areGpg45VcsCorrelated);
             if (profileMatchResponse.isPresent()) {
+                // We are re-using an existing Vot so it might not be a GPG45 vot
```

Comment:
I feel it makes more sense to start with targetVot for all types then, on understanding it must be gpg45 (inverse or this if statement), update it to be gpg45 type

Reply:
Logically I kind of agree with that, but practically there are lots of exit points in this method and you'd have to remember to reset the target vot at every one of them except this one which felt a lot more fragile.

Reply:
Tried to think of a way of making it less fragile but think I have to agree with this. It's a shame but practical